bundle/addon.go

```diff
@@ -289,3 +289,36 @@ func (b *Bundle) DeleteAddon(addonID string, orgID string, userID string) (*apis
 	}
 	return &resp.Data, nil
 }
+
+func (b *Bundle) CreateCustomAddon(userID, addonName, name string, orgId, projectId uint64, workspace string,
+	configs map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
use struct instead of parameters

Reply:
may be apistructs.CustomAddonCreateRequest, but it has too may feilds

Reply:
It is recommended to use apistructs.CustomAddonCreateRequest, which is better than writing so many params.